src/Glpi/Exception/Access/RequiresHttpsException.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * ---------------------------------------------------------------------
+ *
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ *
+ * http://glpi-project.org
+ *
+ * @copyright 2015-2024 Teclib' and contributors.
+ * @licence   https://www.gnu.org/licenses/gpl-3.0.html
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+namespace Glpi\Exception\Access;
+
+use Glpi\Application\View\TemplateRenderer;
+use Symfony\Component\HttpFoundation\Response;
+
+class RequiresHttpsException extends AccessException
+{
+    public function asResponse(): Response
+    {
+        $request = $this->getRequest();
+
+        $cnt = TemplateRenderer::getInstance()->render(
+            'pages/https_only.html.twig',
+            [
+                'secured_url' => 'https://' . $request->getHost() . $request->getUri(),
```

Comment:
`Request::getUri()` contains the scheme.

See https://github.com/symfony/http-foundation/blob/133ac043875f59c26c55e79cf074562127cce4d2/Request.php#L1017-L1024

```suggestion
                'secured_url' => 'https://' . $request->getHost() . $request->getRequestUri(),
```